src/main/java/wolox/training/models/Book.java

```diff
@@ -67,6 +68,18 @@ public Book() {
         //Constructor for JPA
     }
 
+    public Book(BookInfoDTO bookInfoDTO) {
+        this.genre = "N/A - External API";
+        this.author = bookInfoDTO.getAuthors().get(0).getName();
+        this.image = bookInfoDTO.getUrl();
```

Comment:
The attribute you need for the image is not the url, check out the `cover` element in the JSON, it has three sizes: small, medium and large. You can pick one of those 🙂 
![image](https://user-images.githubusercontent.com/43755493/105850385-694dd480-5fc0-11eb-95b5-cc701d11c350.png)